botfront/imports/ui/components/nlu/evaluation/EntityReport.jsx

```diff
@@ -344,7 +344,7 @@ export default class EntityReport extends React.Component {
     }
 
     render = () => {
-        const { accuracy, precision, f1_score: f1 } = this.props;
+        const { accuracy = 0, precision = 0, f1_score: f1 = 0 } = this.props;
```

Comment:
Rather than setting these values to 0, better to not load EntityReport when there is no report, here: https://github.com/botfront/botfront/blob/0fa9ce06f91fa2608b58b48fa581db1b0b6d8588/botfront/imports/ui/components/nlu/evaluation/Evaluation.jsx#L74 Seems like `entityEvaluation` is an empty object, we need a check to only render the tab if its properties exist.